src/components/gcodeviewer/Viewer.vue

```diff
@@ -43,11 +43,7 @@
 
 <template>
     <div>
-        <panel
-            :title="$t('GCodeViewer.Title')"
-            :icon="mdiVideo3d"
-            card-class="gcode-viewer-panel"
-            :margin-bottom="false">
+        <panel :title="panelTitle" :icon="mdiVideo3d" card-class="gcode-viewer-panel" :margin-bottom="false">
```

Comment:
```suggestion
        <panel :title="panelTitle.toString()" :icon="mdiVideo3d" card-class="gcode-viewer-panel" :margin-bottom="false">
```
this will prevent a weak IDE warning